calcom/cal.com
# fix: #14875 - Fixed Scrollable overlay AddActionDialog Component

## What does this PR do?

This PR fixes a bug that the UI was breaking in AddActionDialog on reducing the window height 

fix: #14875 

## Type of change
- Bug fix (non-breaking change which fixes an issue)


## How should this be tested?

1.Go to 'Workflow' -> Add/Update a workflow
2.Click on 'Add action'
3.Reduce the browser window height manually or from responsive view in Chrome DevTools and notice that partial information is shown to the user.

## Mandatory Tasks


## Checklist
